src/scene/vcModel.cpp

```diff
@@ -488,9 +488,18 @@ void vcModel::HandleContextMenu(vcState *pProgramState)
   ImGui::Separator();
 
 #if VC_HASCONVERT
-  if (((m_pPreferredProjection == nullptr && pProgramState->gis.SRID == 0) || (m_pPreferredProjection != nullptr && m_pPreferredProjection->srid == pProgramState->gis.SRID)) && (m_defaultMatrix == m_sceneMatrix))
+  if ((m_pPreferredProjection == nullptr && pProgramState->gis.SRID == 0) || (m_pPreferredProjection != nullptr && m_pPreferredProjection->srid == pProgramState->gis.SRID))
   {
-    if (ImGui::BeginMenu(vcString::Get("sceneExplorerExportPointCloud")))
+    bool matrixEqual = true;
+    for (int i = 0; i < m_defaultMatrix.ElementCount; i++)
+    {
+      if (abs(m_defaultMatrix.a[i] - m_sceneMatrix.a[i]) > UD_EPSILON)
```

Comment:
This isn't compiling on Ubuntu 16.04 with Clang, you should use `udAbs` instead.

Reply:
I fixed it, and have moved the codes into udMath.